app/controllers/distributions_controller.rb

```diff
@@ -141,6 +141,7 @@ def edit
       @distribution.line_items.build if @distribution.line_items.size.zero?
       @items = current_organization.items.alphabetized
       @storage_locations = current_organization.storage_locations.active_locations.has_inventory_items.alphabetized
+      @audit_warning = current_organization.audits.where("updated_at > ?", @distribution.created_at).any?
     else
```

Comment:
Can/should we  limit the warning to audits from the storage location that the distribution is associated with?  Or is that going to miss an edge case where they would need an inventory adjustment.

Reply:
Good call - fixed!

Reply:
LGTM!